src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
+    config:set("cluster", "n", integer_to_list(NodeCount)),
```

Comment:
[`config:set_integer/3`](https://github.com/apache/couchdb-config/blob/master/src/config.erl#L68) is better here.